testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6307,6 +6307,22 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     }
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6314">[CALCITE-6314]
+   * Add RANDOM function (enabled in Postgres library)</a>. */
+  @Test void testRandomFunc() {
+    final SqlOperatorFixture f = fixture();
+    f.setFor(SqlLibraryOperators.RANDOM, VmName.EXPAND);
+    f.checkFails("^random^", "Column 'RANDOM' not found in any table", false);
+    Consumer<SqlOperatorFixture> test = fixture -> {
```

Comment:
sorry this looks good but can we just call this `consumer` to fit the pattern of other tests? `test` doesn't tell us much

Reply:
Sure, I've renamed test to consumer.